### DISCUSSION TITLE
How do I get this to work in a Maui Library?

### DISCUSSION THREAD

**Discussion by richas32** [CommunityToolkit/Maui#1410] [Q&A] [answered]
trying to include community toolkit.maui and mvvm in a library and use the MCT Expander.  However I keep getting this error
XFC0000	Cannot resolve type "http://schemas.microsoft.com/dotnet/2022/maui/toolkit:mct:Expander".  \Summary\SummaryView.xaml	19	
From what I read this seems like it needs the .UseCommunityToolkit() call in the MauiProgram.  I have this in my main application project but I don't know how to get the Library to pick up this call so that it can recognize the use of the communitytoolkit and resolve the expander.
Can this be used in a library?  Is there a solution or workaround that i'm missing to get this to work?

**Comment by TheCodeTraveler** [comment, marked as answer]
Here's the answer I posted on StackOverflow:

https://stackoverflow.com/a/76452761/5953643
